### PR TITLE
Add resource manager documentation

### PR DESCRIPTION
**Pull request by wvandeun**
TODO:
- [x] SDK
- [x] Tutorial

### CODE REVIEW

**Comment by wvandeun** [comment]
> This PR is probably a good opportunity to remove limitations mentioned [here](https://github.com/opsmill/infrahub/blob/develop/docs/docs/topics/ipam.mdx?plain=1#L96-L101).

Done!

**Review by dgarros — APPROVED** [review]
👍🏼

File: docs/docs/guides/resource-manager.mdx [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+        label: Name
+        optional: false
+    relationships:
+      - name: primary_ip
+        label: "Primary IP Address"
+        peer: IpamIPAddress
+        optional: false
+        kind: Attribute
+        cardinality: one
+```
```
Comment (ogenstad) [inline docs/docs/guides/resource-manager.mdx:52]: Perhaps add a "note" or "warning" that this schema might conflict with the other demo schema if that has been loaded?

**Review by ogenstad — APPROVED** [review]

File: docs/docs/guides/resource-manager.mdx (hunk 1 above)
Comment (wvandeun) [reply docs/docs/guides/resource-manager.mdx:52]: The guide mentions this near the top of the page
image

**Pull request merged**